onadata/apps/api/tasks.py

```diff
@@ -106,3 +110,19 @@ def send_verification_email(email, message_txt, subject):
 def send_account_lockout_email(email, message_txt, subject):
     """Sends account locked email."""
     send_generic_email(email, message_txt, subject)
+
+
+@app.task()
+def delete_inactive_submissions():
+    """
+    Task to periodically delete soft deleted submissions from db
+    """
+    submissions_lifespan = getattr(settings, "INACTIVE_SUBMISSIONS_LIFESPAN", 365)
```

Comment:
This line seems to define a hard limit of 365 days if the setting does not exist. Could we take the option of simply not taking any action if the setting is not defined/set? That will maintain the status quo as is now instead of on deployment all deleted items older than a year being removed. For some installations, this may not be ideal and prior communication may be necessary.